### ISSUE TITLE
Missing export default

### ISSUE THREAD

**Issue by nardeas** [gajus/roarr#6]
With Node 9.3 doing

```javascript
import log from 'roarr';
log('foo')
```

results in `TypeError: log is not a function`. I think it's because `roarr` is missing the default export:

```javascript
console.log(log)
```

outputs

```
{ default: 
   { [Function: log]
     child: [Function],
     debug: [Function],
     error: [Function],
     fatal: [Function],
     info: [Function],
     trace: [Function],
     warn: [Function] } }
```

**Comment by normancarcamo** [comment]
I think it is intended to be used with ES6+, I have experimented the same issue but it works if I select the default object.

My solution was:
```js
const log = require('roarr').default;

log('foo');

// {"context":{},"message":"foo","sequence":0,"time":1533646976588,"version":"1.0.0"}
```

**Comment by gajus** [comment]
> My solution was:

That is the correct solution.

**Comment by fredericgermain** [comment]
This fails with node 12 experimental ESM modules support too. It seems they are stricter with CJS modules, and handling of module.exports.default, which seems to be is a babel thing.
Could it be possible to add `add-module-exports` to .babelrc so we don't have to do this .default? Or maybe at some point expose the ESM module as well (that is, the src/ without flow types...)?

`cat index.mjs`

```
import roarr from 'roarr';

console.log(roarr);
```
`node --experimental-modules index.mjs` 
```
(node:5139) ExperimentalWarning: The ESM module loader is experimental.
{ default:
   { [Function: log]
     child: [Function],
     getContext: [Function],
     debug: [Function],
     error: [Function],
     fatal: [Function],
     info: [Function],
     trace: [Function],
     warn: [Function] } }
```

**Comment by gajus** [comment]
Fixed in v7.0.0.

Now you should import `Roarr`, i.e.

```js
import {Roarr} from 'roarr';

Roarr.log('foo');

```
